docs/pages/user-guides.mdx

```diff
@@ -0,0 +1,225 @@
+---
+title: User Guides
+description: Documentation for end users
+h1: End User Documentation
+---
+
+Teleport is a complex security solution, and most of our documentation covers
+the installation, configuration, and management of Teleport for administrators.
+For end users, this page covers the basics of authenticating to Teleport and
+accessing resources.
+
+## tsh
+
+`tsh` lets you authenticate to Teleport, list services, and initate either
+direct connections or proxy tunnels to them. After [downloading](/downloads)
+and installing `tsh`, sign in to your Teleport plane:
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+```code
+$ tsh login --proxy=teleport.example.com --user=alice
```

Comment:
All enterprise customers will use a SAML or OIDC connector. I would use one here.

Reply:
Requesting input from @ptgott and @rsyracuse: It may be that I'm being overly pedantic about this, if so please check me at the door.

Reply:
Discussed with the team. I'm making `oss` and `enterprise` discrete scoped blocks and adding tabs in the Enterprise block for local and IdP users.